src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ActionListExecutorTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Xunit;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Actions;
+using System.Threading;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Exceptions;
+using Microsoft.Diagnostics.Tools.Monitor;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Diagnostics.Monitoring.TestCommon.Options;
+using Xunit.Abstractions;
+using Microsoft.Extensions.Options;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Triggers;
+
+namespace Microsoft.Diagnostics.Monitoring.Tool.UnitTests
+{
+    public sealed class ActionListExecutorTests
+    {
+        private const int TokenTimeoutMs = 10000;
+
+        private ActionListExecutor _executor;
```

Comment:
Because you are now using `TestHostHelper.CreateCollectionRulesHost`, there is no need for this field. Please remove it and change the test methods to just use a local ActionListExecutor.